handbook/company/pricing-features-table.yml

```diff
@@ -15,9 +15,9 @@
     - name: Zero-touch setup for macOS computers
       tier: Premium
       comingSoon: false
-    - name: Safely execute remote scripts
+    - name: Safely execute custom scripts
```

Comment:
Should we specify that this is cross-platform? "(macOS, Windows, Linux)

Reply:
I think that's implied

Reply:
Agreed.

Reply:
I don't think it is. Every item above and below it says macOS.

Reply:
That's fair. Submitted a PR to address this here: https://github.com/fleetdm/fleet/pull/14192